client/rpc/debug/src/lib.rs

```diff
@@ -291,7 +293,7 @@ where
 						.map_err(|e| internal_err(format!("Runtime api access error: {:?}", e)))?
 						.map_err(|e| internal_err(format!("DispatchError: {:?}", e)))?;
 
-						Ok(proxy::v1::Result::V1(proxy::v1::ResultV1::Single(result)))
+						Ok(moonbeam_rpc_primitives_debug::v1::Response::Single(result).into())
```

Comment:
If we are planning to go straight to substitutes, we should not need `v1` and `v2` in the client side.

Reply:
Let's keep v1 for now, and remove it while we move to substitutes.

Reply:
Sure! please update the description with that followup